srcpkgs/git-toolbelt/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'git-toolbelt'
 pkgname=git-toolbelt
-version=1.6.0
+version=1.7.0
 revision=1
 depends="git moreutils fzf"
 short_desc="Helper tools to make everyday life with Git much easier"
 maintainer="Daniel Lewan <vision360.daniel@gmail.com>"
 license="BSD-3-Clause"
 homepage="https://github.com/nvie/git-toolbelt"
 distfiles="https://github.com/nvie/git-toolbelt/archive/v${version}.tar.gz"
-checksum=@8b0d689fcbda5148d0c8728b542fb691e7bbb91ba438f57db67e7f740819e526
+checksum=@8c54c3232b2fa1d54ffe1705bd8664f17d0a998140a08de45696df7e8bde8708
```

Comment:
Please switch to a normal checksum (without `@`), GH makes deterministic tarballs now.

Reply:
Done